AutoRest/AutoRest.psm1

```diff
@@ -3,6 +3,7 @@ $script:ModuleVersion = (Import-PowerShellDataFile -Path "$($script:ModuleRoot)\
 
 # Detect whether at some level dotsourcing was enforced
 $script:doDotSource = Get-PSFConfigValue -FullName AutoRest.Import.DoDotSource -Fallback $false
+[PSFramework.Message.MessageLevel] $script:logLevel = Get-PSFConfigValue -FullName AutoRest.Logging.Level -Fallback "Warning"
```

Comment:
Sorry, but please do not modify the psm1 file or use a configuration setting pre-initialization (the two settings related to the import sequence are by necessity an exemption to that rule).

Furthermore, reading the config on module import makes later changes to the setting ineffective, when the user should be able to just change the setting for the current process and get results - thus I'd argue to place the `Get-PSFConfigValue` call into the begin block of ConvertFrom-ARSwagger (also don't think it needs to be script-scope in that case)

Reply:
No need to be sorry. I don't know what I am doing with your config framework. I might as well just hardcoded the default value here, because there was no Set-PSFConfigValue anyway. Feel free to push into my branch with fixed code if you feel like it :) I might not get back to this until next week. :)